### ISSUE TITLE
If the text length is larger than 14500 then the <HTMLView> is not rendering the html text

### ISSUE THREAD

**Issue by tanviag** [jsdf/react-native-htmlview#189]

**Comment by isilher** [comment]
This is most likely because of the way large Text components render on iOS. You can solve it by adding `{{ flex: 1 }}` to all component styles passed through with the `stylesheet` prop. (see also: https://stackoverflow.com/questions/35992201/react-native-text-component-not-render-anything-if-text-string-is-large)

The only problem I have (with large Text components in general) is that I can't get this to work in a ScrollView. (which is most likely the use-case, since large texts won't fit in one screen). I'll update if I find a solution to it.

**Comment by tanviag** [comment]
Yes , the Text is in Scrollview and that's why the problem occurs.

**Comment by isilher** [comment]
Relates to: https://stackoverflow.com/questions/41001693/large-amount-of-text-not-rendering-properly-in-a-scroll-view-react-native

**Comment by rogerkerse** [comment]
I think this is the issue this library could solve as well.

**Comment by huy-lv** [comment]
Hey, how to fix this problem?
